help: Update mobile instructions for Flutter launch

laurynmm opened the pull request:
Part of https://github.com/zulip/zulip-flutter/issues/715 and https://github.com/zulip/zulip/issues/34748:

> Update the Help Center for changes to the app (try to think of things; also look through open post-launch issues for things that are features in the legacy app; also grep through the Help Center source for "mobile" tabs)

**Notes**:
- These changes are for help articles with "{tab|mobile}" instructions that will be **updated** for the Flutter launch.
  - Help articles that will only have current mobile tab content deleted will be in a separate pull request to help with the review/edit process.
- There are a few icons used in the Flutter app that are not in the shared icons in this repository. They are added in the first three commits of these changes.
  - The menu icon used in the bottom bar of the app
  - The switch account icon used in the menu action sheet of the app
  - The camera and image icons used in the compose box of the app

<details>
<summary>Diff of changes to rendered help center</summary>

```diff
diff -r -u current_help_html/combined-feed.html new_help_html/combined-feed.html
--- current_help_html/combined-feed.html	2025-06-11 11:40:52.163967707 +0200
+++ new_help_html/combined-feed.html	2025-06-11 12:22:24.946904888 +0200
@@ -466,8 +466,8 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>Tap the <strong>Combined feed</strong>
-   (<img src="/static/images/help/mobile-globe-icon.svg" alt="globe" class="help-center-icon"/>)
-   tab in the upper left corner of the app.</li>
+   (<i class="zulip-icon mobile-help zulip-icon-all-messages"></i>)
+   tab at the bottom of the app.</li>
 </ol>
 <p></div>
 </div>
diff -r -u current_help_html/configure-how-links-open.html new_help_html/configure-how-links-open.html
--- current_help_html/configure-how-links-open.html	2025-06-11 11:40:58.823967539 +0200
+++ new_help_html/configure-how-links-open.html	2025-06-11 12:22:31.678904718 +0200
@@ -455,10 +455,11 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap your <strong>profile picture</strong> in the bottom right corner of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
 </li>
 <li>
-<p>Tap <strong>Settings</strong>.</p>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-gear"></i> <strong>Settings</strong>.</p>
 </li>
 <li>
 <p>Toggle <strong>Open links with in-app browser</strong>.</p>
diff -r -u current_help_html/dark-theme.html new_help_html/dark-theme.html
--- current_help_html/dark-theme.html	2025-06-11 11:40:46.785967843 +0200
+++ new_help_html/dark-theme.html	2025-06-11 12:22:19.416905027 +0200
@@ -467,13 +467,14 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap your <strong>profile picture</strong> in the bottom right corner of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
 </li>
 <li>
-<p>Tap <strong>Settings</strong>.</p>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-gear"></i> <strong>Settings</strong>.</p>
 </li>
 <li>
-<p>Toggle <strong>Night mode</strong>.</p>
+<p>Select the desired theme.</p>
 </li>
 </ol>
 <p></div>
diff -r -u current_help_html/delete-a-message.html new_help_html/delete-a-message.html
--- current_help_html/delete-a-message.html	2025-06-11 11:40:51.354967727 +0200
+++ new_help_html/delete-a-message.html	2025-06-11 12:22:24.118904909 +0200
@@ -488,9 +488,14 @@
 <p>Press and hold a message until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Delete message</strong> to delete the content of the message.  If you do not
-   see the <strong>Delete message</strong> option, you do not have permission to delete the
-   content of this message.</p>
+<p>Tap <strong>Edit message</strong>. If you do not see the <strong>Edit message</strong> option,
+   you do not have permission to change the content of this message.</p>
+</li>
+<li>
+<p>Delete the content of the message.</p>
+</li>
+<li>
+<p>Tap <strong>Save</strong>.</p>
 </li>
 </ol>
 <p></div>
diff -r -u current_help_html/direct-messages.html new_help_html/direct-messages.html
--- current_help_html/direct-messages.html	2025-06-11 11:40:54.974967636 +0200
+++ new_help_html/direct-messages.html	2025-06-11 12:22:27.717904818 +0200
@@ -495,12 +495,11 @@
 <ol>
 <li>
 <p>Tap the <strong>Direct messages</strong>
-   ( <img src="/static/images/help/mobile-dm-tab-icon.svg" alt="direct messages" class="help-center-icon"/> )
-   tab at the bottom of the app.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-user"></i>) tab at the
+   bottom of the app.</p>
 </li>
 <li>
-<p>Tap the <strong>New DM</strong> button at the top of the app to message one person, or
-   <strong>New group DM</strong> to message any number of people.</p>
+<p>Tap the <strong>New DM</strong> button.</p>
 </li>
 <li>
 <p>Start typing the name of the person you want to message, and
@@ -508,14 +507,12 @@
 </li>
 <li>
 <p><em>(if sending a group DM)</em> Continue adding as many message recipients as you
-   like. Approve by tapping the <strong>checkmark</strong> (<img
-   src="/static/images/help/mobile-check-circle-icon.svg" alt="checkmark"
-   class="help-center-icon"/>) button in the bottom right corner of the app.</p>
+   like. Approve by tapping the <strong>Next</strong> button in the top right corner of the app.</p>
 </li>
 <li>
-<p>Compose your message, and tap the <strong>send</strong>
-   (<img src="/static/images/help/mobile-send-circle-icon.svg" alt="send" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+<p>Compose your message, and tap the <strong>Send</strong>
+   (<i class="zulip-icon mobile-help zulip-icon-send"></i>) button in the
+   bottom right corner of the app.</p>
 </li>
 </ol>
 <div class="admonition tip">
@@ -526,14 +523,14 @@
 </div>
 </div></p>
 <h2 id="view-your-dms">View your DMs</h2>
-<p>There are a few different ways to view your DM conversations in the web and
-desktop apps.</p>
+<p>There are a few different ways to view your DM conversations.</p>
 <p><div class="tabbed-section has-tabs" markdown="1">
 <ul class="nav">
 <li data-tab-key="via-left-sidebar" tabindex="0">Via left sidebar</li>
 <li data-tab-key="via-right-sidebar" tabindex="0">Via right sidebar</li>
 <li data-tab-key="via-search-box" tabindex="0">Via search box</li>
 <li data-tab-key="via-compose-box" tabindex="0">Via compose box</li>
+<li data-tab-key="mobile" tabindex="0">Mobile</li>
 </ul>
 <div class="blocks">
 <div class="tab-content" data-tab-key="via-left-sidebar" markdown="1"></p>
@@ -608,6 +605,18 @@
 </li>
 </ol>
 <p></div>
+<div class="tab-content" data-tab-key="mobile" markdown="1"></p>
+<ol>
+<li>
+<p>Tap the <strong>Direct messages</strong>
+   (<i class="zulip-icon mobile-help zulip-icon-user"></i>) tab at the
+   bottom of the app.</p>
+</li>
+<li>
+<p>Tap a recent DM conversation to view it.</p>
+</li>
+</ol>
+<p></div>
 </div>
 </div></p>
 <h2 id="find-a-direct-message-conversation">Find a direct message conversation</h2>
diff -r -u current_help_html/edit-a-message.html new_help_html/edit-a-message.html
--- current_help_html/edit-a-message.html	2025-06-11 11:40:51.242967730 +0200
+++ new_help_html/edit-a-message.html	2025-06-11 12:22:24.003904911 +0200
@@ -490,9 +490,7 @@
 <p>Edit the content of the message.</p>
 </li>
 <li>
-<p>Approve by tapping the <strong>checkmark</strong>
-   (<img src="/static/images/help/mobile-check-circle-icon.svg" alt="checkmark" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+<p>Tap <strong>Save</strong>.</p>
 </li>
 </ol>
 <p></div>
diff -r -u current_help_html/emoji-reactions.html new_help_html/emoji-reactions.html
--- current_help_html/emoji-reactions.html	2025-06-11 11:40:52.977967687 +0200
+++ new_help_html/emoji-reactions.html	2025-06-11 12:22:25.751904867 +0200
@@ -488,7 +488,8 @@
 <p>Press and hold a message until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Add a reaction</strong>.</p>
+<p>Select one of the emojis at the top of the menu, or tap <strong>more</strong>
+   (<i class="zulip-icon mobile-help zulip-icon-chevron-right"></i>).</p>
 </li>
 <li>
 <p>Start typing the name of the emoji you want to use, and select an emoji from
diff -r -u current_help_html/finding-a-conversation-to-read.html new_help_html/finding-a-conversation-to-read.html
--- current_help_html/finding-a-conversation-to-read.html	2025-06-11 11:40:43.570967924 +0200
+++ new_help_html/finding-a-conversation-to-read.html	2025-06-11 12:22:16.601905098 +0200
@@ -472,39 +472,30 @@
    or desktop app) to select the next conversation.</p>
 </li>
 </ol>
-<div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by clicking the
-<strong>collapse</strong>
-(<i class="zulip-icon zulip-icon-arrow-down"></i>)
-or <strong>expand</strong>
-(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
-icon to the left of a channel name.</p>
-</div>
 <p></div>
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Inbox</strong>
-   (<img src="/static/images/help/mobile-inbox-icon.svg" alt="inbox" class="help-center-icon"/>)
+<p>Tap the <strong>Inbox</strong> (<i class="zulip-icon mobile-help zulip-icon-inbox"></i>)
    tab in the bottom left corner of the app.</p>
 </li>
 <li>
 <p>Tap on a conversation you're interested in to view it. You can return to
-   <strong>Inbox</strong> when done (e.g., by using the *<em>back</em> button) to select the next
-   conversation.</p>
+   your <strong>inbox</strong>  when done (e.g., by using the <strong>back</strong> button) to select
+   the next conversation.</p>
 </li>
 </ol>
+<p></div>
+</div>
+</div></p>
 <div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by tapping the
+<p>You can collapse or expand the list of topics in a channel by clicking the
 <strong>collapse</strong>
-(<img src="/static/images/help/mobile-expand-less-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down"></i>)
 or <strong>expand</strong>
-(<img src="/static/images/help/mobile-expand-more-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
 icon to the left of a channel name.</p>
 </div>
-<p></div>
-</div>
-</div></p>
 <h2 id="from-recent-conversations">From Recent conversations</h2>
 <p>Use the <strong>recent conversations</strong> view to get
 an overview of all the ongoing conversations. This view is particularly useful
diff -r -u current_help_html/getting-started-with-zulip.html new_help_html/getting-started-with-zulip.html
--- current_help_html/getting-started-with-zulip.html	2025-06-11 11:40:42.131967960 +0200
+++ new_help_html/getting-started-with-zulip.html	2025-06-11 12:22:15.222905133 +0200
@@ -535,39 +535,30 @@
    or desktop app) to select the next conversation.</p>
 </li>
 </ol>
-<div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by clicking the
-<strong>collapse</strong>
-(<i class="zulip-icon zulip-icon-arrow-down"></i>)
-or <strong>expand</strong>
-(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
-icon to the left of a channel name.</p>
-</div>
 <p></div>
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Inbox</strong>
-   (<img src="/static/images/help/mobile-inbox-icon.svg" alt="inbox" class="help-center-icon"/>)
+<p>Tap the <strong>Inbox</strong> (<i class="zulip-icon mobile-help zulip-icon-inbox"></i>)
    tab in the bottom left corner of the app.</p>
 </li>
 <li>
 <p>Tap on a conversation you're interested in to view it. You can return to
-   <strong>Inbox</strong> when done (e.g., by using the *<em>back</em> button) to select the next
-   conversation.</p>
+   your <strong>inbox</strong>  when done (e.g., by using the <strong>back</strong> button) to select
+   the next conversation.</p>
 </li>
 </ol>
+<p></div>
+</div>
+</div></p>
 <div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by tapping the
+<p>You can collapse or expand the list of topics in a channel by clicking the
 <strong>collapse</strong>
-(<img src="/static/images/help/mobile-expand-less-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down"></i>)
 or <strong>expand</strong>
-(<img src="/static/images/help/mobile-expand-more-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
 icon to the left of a channel name.</p>
 </div>
-<p></div>
-</div>
-</div></p>
 <h4 id="from-recent-conversations">From Recent conversations</h4>
 <p>Use the <strong>recent conversations</strong> view to get
 an overview of all the ongoing conversations. This view is particularly useful
@@ -743,8 +734,7 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Channels</strong>
-   (<img src="/static/images/help/mobile-hash-icon.svg" alt="hash" class="help-center-icon"/>)
+<p>Tap the <strong>Channels</strong> (<i class="zulip-icon mobile-help zulip-icon-hash"></i>)
    tab at the bottom of the app.</p>
 </li>
 <li>
@@ -759,8 +749,8 @@
 </li>
 <li>
 <p>Compose your message, and tap the <strong>send</strong>
-   (<img src="/static/images/help/mobile-send-circle-icon.svg" alt="send" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-send"></i>) button in the
+   bottom right corner of the app.</p>
 </li>
 </ol>
 <p></div>
@@ -810,12 +800,11 @@
 <ol>
 <li>
 <p>Tap the <strong>Direct messages</strong>
-   ( <img src="/static/images/help/mobile-dm-tab-icon.svg" alt="direct messages" class="help-center-icon"/> )
-   tab at the bottom of the app.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-user"></i>) tab at the
+   bottom of the app.</p>
 </li>
 <li>
-<p>Tap the <strong>New DM</strong> button at the top of the app to message one person, or
-   <strong>New group DM</strong> to message any number of people.</p>
+<p>Tap the <strong>New DM</strong> button.</p>
 </li>
 <li>
 <p>Start typing the name of the person you want to message, and
@@ -823,14 +812,12 @@
 </li>
 <li>
 <p><em>(if sending a group DM)</em> Continue adding as many message recipients as you
-   like. Approve by tapping the <strong>checkmark</strong> (<img
-   src="/static/images/help/mobile-check-circle-icon.svg" alt="checkmark"
-   class="help-center-icon"/>) button in the bottom right corner of the app.</p>
+   like. Approve by tapping the <strong>Next</strong> button in the top right corner of the app.</p>
 </li>
 <li>
-<p>Compose your message, and tap the <strong>send</strong>
-   (<img src="/static/images/help/mobile-send-circle-icon.svg" alt="send" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+<p>Compose your message, and tap the <strong>Send</strong>
+   (<i class="zulip-icon mobile-help zulip-icon-send"></i>) button in the
+   bottom right corner of the app.</p>
 </li>
 </ol>
 <div class="admonition tip">
diff -r -u current_help_html/inbox.html new_help_html/inbox.html
--- current_help_html/inbox.html	2025-06-11 11:40:51.942967713 +0200
+++ new_help_html/inbox.html	2025-06-11 12:22:24.715904894 +0200
@@ -477,39 +477,30 @@
    or desktop app) to select the next conversation.</p>
 </li>
 </ol>
-<div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by clicking the
-<strong>collapse</strong>
-(<i class="zulip-icon zulip-icon-arrow-down"></i>)
-or <strong>expand</strong>
-(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
-icon to the left of a channel name.</p>
-</div>
 <p></div>
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Inbox</strong>
-   (<img src="/static/images/help/mobile-inbox-icon.svg" alt="inbox" class="help-center-icon"/>)
+<p>Tap the <strong>Inbox</strong> (<i class="zulip-icon mobile-help zulip-icon-inbox"></i>)
    tab in the bottom left corner of the app.</p>
 </li>
 <li>
 <p>Tap on a conversation you're interested in to view it. You can return to
-   <strong>Inbox</strong> when done (e.g., by using the *<em>back</em> button) to select the next
-   conversation.</p>
+   your <strong>inbox</strong>  when done (e.g., by using the <strong>back</strong> button) to select
+   the next conversation.</p>
 </li>
 </ol>
+<p></div>
+</div>
+</div></p>
 <div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by tapping the
+<p>You can collapse or expand the list of topics in a channel by clicking the
 <strong>collapse</strong>
-(<img src="/static/images/help/mobile-expand-less-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down"></i>)
 or <strong>expand</strong>
-(<img src="/static/images/help/mobile-expand-more-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
 icon to the left of a channel name.</p>
 </div>
-<p></div>
-</div>
-</div></p>
 <div class="admonition keyboard_tip">
 <p>The arrow keys and vim navigation keys (<kbd>J</kbd>, <kbd>K</kbd>,
 <kbd>L</kbd>, <kbd>H</kbd>) can be used to move between elements.</p>
diff -r -u current_help_html/introduction-to-topics#how-to-start-a-new-topic.html new_help_html/introduction-to-topics#how-to-start-a-new-topic.html
--- current_help_html/introduction-to-topics#how-to-start-a-new-topic.html	2025-06-11 11:40:43.455967927 +0200
+++ new_help_html/introduction-to-topics#how-to-start-a-new-topic.html	2025-06-11 12:22:16.485905101 +0200
@@ -546,8 +546,7 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Channels</strong>
-   (<img src="/static/images/help/mobile-hash-icon.svg" alt="hash" class="help-center-icon"/>)
+<p>Tap the <strong>Channels</strong> (<i class="zulip-icon mobile-help zulip-icon-hash"></i>)
    tab at the bottom of the app.</p>
 </li>
 <li>
@@ -562,8 +561,8 @@
 </li>
 <li>
 <p>Compose your message, and tap the <strong>send</strong>
-   (<img src="/static/images/help/mobile-send-circle-icon.svg" alt="send" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-send"></i>) button in the
+   bottom right corner of the app.</p>
 </li>
 </ol>
 <p></div>
diff -r -u current_help_html/introduction-to-topics.html new_help_html/introduction-to-topics.html
--- current_help_html/introduction-to-topics.html	2025-06-11 11:40:43.334967930 +0200
+++ new_help_html/introduction-to-topics.html	2025-06-11 12:22:16.365905104 +0200
@@ -546,8 +546,7 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Channels</strong>
-   (<img src="/static/images/help/mobile-hash-icon.svg" alt="hash" class="help-center-icon"/>)
+<p>Tap the <strong>Channels</strong> (<i class="zulip-icon mobile-help zulip-icon-hash"></i>)
    tab at the bottom of the app.</p>
 </li>
 <li>
@@ -562,8 +561,8 @@
 </li>
 <li>
 <p>Compose your message, and tap the <strong>send</strong>
-   (<img src="/static/images/help/mobile-send-circle-icon.svg" alt="send" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-send"></i>) button in the
+   bottom right corner of the app.</p>
 </li>
 </ol>
 <p></div>
diff -r -u current_help_html/logging-in.html new_help_html/logging-in.html
--- current_help_html/logging-in.html	2025-06-11 11:40:46.106967860 +0200
+++ new_help_html/logging-in.html	2025-06-11 12:22:18.795905043 +0200
@@ -562,16 +562,18 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap your <strong>profile picture</strong> in the bottom right corner of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
 </li>
 <li>
-<p>Tap <strong>Switch account</strong>.</p>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-mobile-arrow-left-right"></i>
+   <strong>Switch account</strong>.</p>
 </li>
 <li>
 <p>Tap <strong>Add new account</strong>.</p>
 </li>
 <li>
-<p>Enter the Zulip URL of the organization, and tap <strong>Enter</strong>.</p>
+<p>Enter the Zulip URL of the organization, and tap <strong>Continue</strong>.</p>
 </li>
 <li>
 <p>Follow the on-screen instructions.</p>
@@ -606,10 +608,12 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap your <strong>profile picture</strong> in the bottom right corner of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
 </li>
 <li>
-<p>Tap <strong>Switch account</strong>.</p>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-mobile-arrow-left-right"></i>
+   <strong>Switch account</strong>.</p>
 </li>
 <li>
 <p>Tap on the desired Zulip organization.</p>
diff -r -u current_help_html/logging-out.html new_help_html/logging-out.html
--- current_help_html/logging-out.html	2025-06-11 11:40:46.212967857 +0200
+++ new_help_html/logging-out.html	2025-06-11 12:22:18.890905040 +0200
@@ -464,7 +464,15 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap your <strong>profile picture</strong> in the bottom right corner of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
+</li>
+<li>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-mobile-arrow-left-right"></i>
+   <strong>Switch account</strong>.</p>
+</li>
+<li>
+<p>Tap on the Zulip organization you want to log out of.</p>
 </li>
 <li>
 <p>Tap <strong>Log out</strong>.</p>
diff -r -u current_help_html/marking-messages-as-read.html new_help_html/marking-messages-as-read.html
--- current_help_html/marking-messages-as-read.html	2025-06-11 11:40:52.615967696 +0200
+++ new_help_html/marking-messages-as-read.html	2025-06-11 12:22:25.426904876 +0200
@@ -603,17 +603,16 @@
 <ol>
 <li>
 <p>Tap a channel, topic, or the <strong>Combined feed</strong>
-   (<img src="/static/images/help/mobile-globe-icon.svg" alt="globe" class="help-center-icon"/>)
-   tab.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-all-messages"></i>) tab.</p>
 </li>
 <li>
-<p>Tap <strong>Mark channel as read</strong>, <strong>Mark topic as read</strong>, or <strong>Mark all as read</strong>
-   near the top right corner of the app.</p>
+<p>Scroll down to the bottom of the message view, and tap <strong>Mark all messages
+   as read</strong>.</p>
 </li>
 </ol>
 <div class="admonition tip">
-<p>You can also press and hold a topic until the long-press menu appears, and
-select <strong>Mark topic as read</strong>.</p>
+<p>You can also press and hold a channel or topic until the long-press menu
+appears, and select the option to mark as read.</p>
 </div>
 <p></div>
 </div>
diff -r -u current_help_html/mute-a-topic.html new_help_html/mute-a-topic.html
--- current_help_html/mute-a-topic.html	2025-06-11 11:40:57.721967567 +0200
+++ new_help_html/mute-a-topic.html	2025-06-11 12:22:30.773904741 +0200
@@ -558,23 +558,20 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Channels</strong>
-   (<img src="/static/images/help/mobile-hash-icon.svg" alt="hash" class="help-center-icon"/>)
+<p>Tap the <strong>Channels</strong> (<i class="zulip-icon mobile-help zulip-icon-hash"></i>)
    tab at the bottom of the app.</p>
 </li>
 <li>
 <p>Select a channel containing topics you want to configure.</p>
 </li>
 <li>
-<p>Tap the <strong>Topics list</strong>
-   (<img src="/static/images/help/mobile-list-icon.svg" alt="list" class="help-center-icon"/>)
-   icon in the upper right corner of the app.</p>
+<p>Tap <strong>Topics</strong> in the upper right corner of the app.</p>
 </li>
 <li>
 <p>Press and hold a topic until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Mute topic</strong> or <strong>Unmute topic</strong>.</p>
+<p>Tap <strong>Mute topic</strong>, <strong>Unmute topic</strong>, or <strong>Follow topic</strong>.</p>
 </li>
 </ol>
 <div class="admonition tip">
diff -r -u current_help_html/quote-or-forward-a-message.html new_help_html/quote-or-forward-a-message.html
--- current_help_html/quote-or-forward-a-message.html	2025-06-11 11:40:48.496967799 +0200
+++ new_help_html/quote-or-forward-a-message.html	2025-06-11 12:22:21.107904984 +0200
@@ -492,10 +492,14 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
+<p>If you're in not in the topic view of the message you want to
+   quote, tap the topic name.</p>
+</li>
+<li>
 <p>Press and hold a message until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Quote message</strong>.</p>
+<p>Tap <strong>Quote and reply</strong>.</p>
 </li>
 <li>
 <p><em>(optional)</em> Delete any parts of the quoted message that are not
@@ -548,18 +552,24 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
+<p>If you're not in the channel view of the message you want to forward,
+   tap the channel name in the message feed or tap the <strong>Channel feed</strong>
+   <i class="zulip-icon mobile-help zulip-icon-all-messages"></i> icon
+   in the bar at the top of the app.</p>
+</li>
+<li>
 <p>Press and hold a message until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Quote message</strong>.</p>
+<p>Tap <strong>Quote and reply</strong>.</p>
 </li>
 <li>
 <p><em>(optional)</em> Delete any parts of the quoted message that you don't want to
    forward.</p>
 </li>
 <li>
-<p>If you are in a channel view, choose a destination topic by tapping the
-   compose box and selecting an existing topic or typing a new topic name.</p>
+<p>Choose a destination topic by tapping the compose box and selecting an
+   existing topic or typing a new topic name.</p>
 </li>
 <li>
 <p>Send your message.</p>
diff -r -u current_help_html/reading-strategies.html new_help_html/reading-strategies.html
--- current_help_html/reading-strategies.html	2025-06-11 11:40:51.833967715 +0200
+++ new_help_html/reading-strategies.html	2025-06-11 12:22:24.600904896 +0200
@@ -480,39 +480,30 @@
    or desktop app) to select the next conversation.</p>
 </li>
 </ol>
-<div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by clicking the
-<strong>collapse</strong>
-(<i class="zulip-icon zulip-icon-arrow-down"></i>)
-or <strong>expand</strong>
-(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
-icon to the left of a channel name.</p>
-</div>
 <p></div>
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Inbox</strong>
-   (<img src="/static/images/help/mobile-inbox-icon.svg" alt="inbox" class="help-center-icon"/>)
+<p>Tap the <strong>Inbox</strong> (<i class="zulip-icon mobile-help zulip-icon-inbox"></i>)
    tab in the bottom left corner of the app.</p>
 </li>
 <li>
 <p>Tap on a conversation you're interested in to view it. You can return to
-   <strong>Inbox</strong> when done (e.g., by using the *<em>back</em> button) to select the next
-   conversation.</p>
+   your <strong>inbox</strong>  when done (e.g., by using the <strong>back</strong> button) to select
+   the next conversation.</p>
 </li>
 </ol>
+<p></div>
+</div>
+</div></p>
 <div class="admonition tip">
-<p>You can collapse or expand the list of topics in a channel by tapping the
+<p>You can collapse or expand the list of topics in a channel by clicking the
 <strong>collapse</strong>
-(<img src="/static/images/help/mobile-expand-less-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down"></i>)
 or <strong>expand</strong>
-(<img src="/static/images/help/mobile-expand-more-icon.svg" alt="inbox" class="help-center-icon"/>)
+(<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
 icon to the left of a channel name.</p>
 </div>
-<p></div>
-</div>
-</div></p>
 <h4 id="from-recent-conversations">From Recent conversations</h4>
 <p>Use the <strong>recent conversations</strong> view to get
 an overview of all the ongoing conversations. This view is particularly useful
@@ -659,8 +650,8 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>Tap the <strong>Combined feed</strong>
-   (<img src="/static/images/help/mobile-globe-icon.svg" alt="globe" class="help-center-icon"/>)
-   tab in the upper left corner of the app.</li>
+   (<i class="zulip-icon mobile-help zulip-icon-all-messages"></i>)
+   tab at the bottom of the app.</li>
 </ol>
 <p></div>
 </div>
@@ -758,9 +749,11 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Mentions</strong>
-   (<img src="/static/images/help/mobile-at-sign-icon.svg" alt="at-sign" class="help-center-icon"/>)
-   tab at the top of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
+</li>
+<li>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-at-sign"></i> <strong>Mentions</strong>.</p>
 </li>
 <li>
 <p>Browse your mentions. You can tap on a message recipient bar to go
@@ -795,9 +788,13 @@
 <p></div>
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
-<li>Tap the <strong>Starred messages</strong>
-   (<img src="/static/images/help/mobile-star-icon.svg" alt="star" class="help-center-icon"/>)
-   tab at the top of the app.</li>
+<li>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
+</li>
+<li>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-star"></i> <strong>Starred messages</strong>.</p>
+</li>
 </ol>
 <p></div>
 </div>
diff -r -u current_help_html/resolve-a-topic.html new_help_html/resolve-a-topic.html
--- current_help_html/resolve-a-topic.html	2025-06-11 11:40:56.708967593 +0200
+++ new_help_html/resolve-a-topic.html	2025-06-11 12:22:29.711904768 +0200
@@ -511,7 +511,7 @@
 <p>Press and hold a topic until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Resolve topic</strong>.</p>
+<p>Tap <strong>Mark as resolved</strong>.</p>
 </li>
 </ol>
 <div class="admonition tip">
@@ -555,7 +555,7 @@
 <p>Press and hold a topic until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Unresolve topic</strong>.</p>
+<p>Tap <strong>Mark as unresolved</strong>.</p>
 </li>
 </ol>
 <div class="admonition tip">
diff -r -u current_help_html/share-and-upload-files.html new_help_html/share-and-upload-files.html
--- current_help_html/share-and-upload-files.html	2025-06-11 11:40:48.952967788 +0200
+++ new_help_html/share-and-upload-files.html	2025-06-11 12:22:21.622904971 +0200
@@ -525,9 +525,9 @@
 </li>
 <li>
 <p>Tap the
-   <strong>paperclip</strong> (<img src="/static/images/help/mobile-paperclip-icon.svg" alt="paperclip" class="help-center-icon"/>),
-   <strong>image</strong> (<img src="/static/images/help/mobile-image-icon.svg" alt="image" class="help-center-icon"/>),
-   or <strong>camera</strong> (<img src="/static/images/help/mobile-camera-icon.svg" alt="camera" class="help-center-icon"/>)
+   <strong>paperclip</strong> (<i class="zulip-icon mobile-help zulip-icon-attachment"></i>),
+   <strong>image</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-image"></i>),
+   or <strong>camera</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-camera"></i>)
    button at the bottom of the app to select one or more files. Zulip will
    upload the files, and insert named links using
    <a href="/help/format-your-message-using-markdown#links">Markdown formatting</a>:
diff -r -u current_help_html/star-a-message.html new_help_html/star-a-message.html
--- current_help_html/star-a-message.html	2025-06-11 11:40:53.210967681 +0200
+++ new_help_html/star-a-message.html	2025-06-11 12:22:26.000904861 +0200
@@ -447,6 +447,8 @@
 <p>Starring messages is a good way to keep track of important messages, such as
 tasks you need to go back to or documents you reference often.</p>
 <h2 id="star-a-message_1">Star a message</h2>
+<p>Starred messages have a filled in star (<i class="zulip-icon zulip-icon-star-filled"></i>)
+to their right.</p>
 <p><div class="tabbed-section has-tabs" markdown="1">
 <ul class="nav">
 <li data-tab-key="desktop-web" tabindex="0">Desktop/Web</li>
@@ -463,8 +465,7 @@
 </li>
 </ol>
 <div class="admonition tip">
-<p>Starred messages have a filled in star (<i class="zulip-icon zulip-icon-star-filled"></i>)
-to their right. You can unstar a message using the same instructions used to star it.</p>
+<p>You can unstar a message using the same instructions used to star it.</p>
 </div>
 <div class="admonition keyboard_tip">
 <p>You can use <kbd>Ctrl</kbd> + <kbd>S</kbd> to star or unstar the selected
@@ -481,8 +482,7 @@
 </li>
 </ol>
 <div class="admonition tip">
-<p>Starred messages have a "starred" label in the bottom right corner of the
-message. You can unstar a message by tapping <strong>Unstar message</strong> in the
+<p>You can unstar a message by tapping <strong>Unstar message</strong> in the
 long-press menu.</p>
 </div>
 <p></div>
@@ -511,9 +511,13 @@
 <p></div>
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
-<li>Tap the <strong>Starred messages</strong>
-   (<img src="/static/images/help/mobile-star-icon.svg" alt="star" class="help-center-icon"/>)
-   tab at the top of the app.</li>
+<li>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
+</li>
+<li>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-star"></i> <strong>Starred messages</strong>.</p>
+</li>
 </ol>
 <p></div>
 </div>
diff -r -u current_help_html/starting-a-new-direct-message.html new_help_html/starting-a-new-direct-message.html
--- current_help_html/starting-a-new-direct-message.html	2025-06-11 11:40:43.817967917 +0200
+++ new_help_html/starting-a-new-direct-message.html	2025-06-11 12:22:16.822905092 +0200
@@ -487,12 +487,11 @@
 <ol>
 <li>
 <p>Tap the <strong>Direct messages</strong>
-   ( <img src="/static/images/help/mobile-dm-tab-icon.svg" alt="direct messages" class="help-center-icon"/> )
-   tab at the bottom of the app.</p>
+   (<i class="zulip-icon mobile-help zulip-icon-user"></i>) tab at the
+   bottom of the app.</p>
 </li>
 <li>
-<p>Tap the <strong>New DM</strong> button at the top of the app to message one person, or
-   <strong>New group DM</strong> to message any number of people.</p>
+<p>Tap the <strong>New DM</strong> button.</p>
 </li>
 <li>
 <p>Start typing the name of the person you want to message, and
@@ -500,14 +499,12 @@
 </li>
 <li>
 <p><em>(if sending a group DM)</em> Continue adding as many message recipients as you
-   like. Approve by tapping the <strong>checkmark</strong> (<img
-   src="/static/images/help/mobile-check-circle-icon.svg" alt="checkmark"
-   class="help-center-icon"/>) button in the bottom right corner of the app.</p>
+   like. Approve by tapping the <strong>Next</strong> button in the top right corner of the app.</p>
 </li>
 <li>
-<p>Compose your message, and tap the <strong>send</strong>
-   (<img src="/static/images/help/mobile-send-circle-icon.svg" alt="send" class="help-center-icon"/>)
-   button in the bottom right corner of the app.</p>
+<p>Compose your message, and tap the <strong>Send</strong>
+   (<i class="zulip-icon mobile-help zulip-icon-send"></i>) button in the
+   bottom right corner of the app.</p>
 </li>
 </ol>
 <div class="admonition tip">
diff -r -u current_help_html/switching-between-organizations.html new_help_html/switching-between-organizations.html
--- current_help_html/switching-between-organizations.html	2025-06-11 11:40:46.322967854 +0200
+++ new_help_html/switching-between-organizations.html	2025-06-11 12:22:18.986905038 +0200
@@ -470,10 +470,12 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap your <strong>profile picture</strong> in the bottom right corner of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
 </li>
 <li>
-<p>Tap <strong>Switch account</strong>.</p>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-mobile-arrow-left-right"></i>
+   <strong>Switch account</strong>.</p>
 </li>
 <li>
 <p>Tap on the desired Zulip organization.</p>
diff -r -u current_help_html/view-images-and-videos.html new_help_html/view-images-and-videos.html
--- current_help_html/view-images-and-videos.html	2025-06-11 11:40:53.352967677 +0200
+++ new_help_html/view-images-and-videos.html	2025-06-11 12:22:26.109904858 +0200
@@ -510,14 +510,14 @@
 <p>Pan the image if you are zoomed in</p>
 </li>
 <li>
-<p>Tap the <strong>ellipsis</strong> (<i class="zulip-icon zulip-icon-more-vertical-spread"></i>)
-  in the bottom right corner of the app to download the image, share it to
+<p>Tap the <strong>copy</strong> (<i class="zulip-icon mobile-help zulip-icon-copy"></i>)
+  icon in the bottom left corner of the app to download the image, share it to
   other apps, or share a link to the image</p>
 </li>
 </ul>
 </li>
 <li>
-<p>Tap the <strong>X</strong> in the upper right corner of the app to close the image viewer
+<p>Tap the <strong>X</strong> in the upper left corner of the app to close the image viewer
    once you are done.</p>
 </li>
 </ol>
diff -r -u current_help_html/view-someones-profile.html new_help_html/view-someones-profile.html
--- current_help_html/view-someones-profile.html	2025-06-11 11:40:54.860967639 +0200
+++ new_help_html/view-someones-profile.html	2025-06-11 12:22:27.551904822 +0200
@@ -478,8 +478,6 @@
 <li>Their name.</li>
 <li>Their email address, if you <a href="/help/configure-email-visibility">have
   permission</a> to view it.</li>
-<li>Their current <a href="/help/change-your-timezone">local time</a>.</li>
-<li>Their <a href="/help/status-and-availability">status and availability</a>.</li>
 <li>Any <a href="/help/custom-profile-fields">custom profile fields</a> they've filled out.</li>
 </ul>
 <p></div>
@@ -516,12 +514,6 @@
 <ol>
 <li>Tap on a user's profile picture or name.</li>
 </ol>
-<div class="admonition tip">
-<p>If you are viewing a direct message conversation, you can also tap the
-<strong>information</strong>
-(<img src="/static/images/help/mobile-info-circle-icon.svg" alt="information" class="help-center-icon"/>)
-button in the top right corner of the app to open their profile.</p>
-</div>
 <p></div>
 </div>
 </div></p>
diff -r -u current_help_html/view-the-markdown-source-of-a-message.html new_help_html/view-the-markdown-source-of-a-message.html
--- current_help_html/view-the-markdown-source-of-a-message.html	2025-06-11 11:40:53.844967665 +0200
+++ new_help_html/view-the-markdown-source-of-a-message.html	2025-06-11 12:22:26.543904847 +0200
@@ -508,7 +508,7 @@
 <p>Press and hold a message until the long-press menu appears.</p>
 </li>
 <li>
-<p>Tap <strong>Copy to clipboard</strong>.</p>
+<p>Tap <strong>Copy message text</strong>.</p>
 </li>
 </ol>
 <p></div>
diff -r -u current_help_html/view-your-mentions.html new_help_html/view-your-mentions.html
--- current_help_html/view-your-mentions.html	2025-06-11 11:40:53.100967683 +0200
+++ new_help_html/view-your-mentions.html	2025-06-11 12:22:25.857904865 +0200
@@ -479,9 +479,11 @@
 <div class="tab-content" data-tab-key="mobile" markdown="1"></p>
 <ol>
 <li>
-<p>Tap the <strong>Mentions</strong>
-   (<img src="/static/images/help/mobile-at-sign-icon.svg" alt="at-sign" class="help-center-icon"/>)
-   tab at the top of the app.</p>
+<p>Tap the <strong>Menu</strong> (<i class="zulip-icon mobile-help zulip-icon-mobile-menu"></i>)
+   tab in the bottom right corner of the app.</p>
+</li>
+<li>
+<p>Tap <i class="zulip-icon mobile-help zulip-icon-at-sign"></i> <strong>Mentions</strong>.</p>
 </li>
 <li>
 <p>Browse your mentions. You can tap on a message recipient bar to go
```
</details>

---

**Screenshots**

[CURRENT DOCUMENTATION](https://zulip.com/help/configure-how-links-open)
![Screenshot from 2025-06-11 11-50-23](https://github.com/user-attachments/assets/fd7d581c-57f9-4281-bcc3-1f366c009d98)

[CURRENT DOCUMENTATION](https://zulip.com/help/switching-between-organizations)
![Screenshot from 2025-06-11 11-52-31](https://github.com/user-attachments/assets/a8347a06-d279-4a4c-8b08-ab5f81e7214c)

[CURRENT DOCUMENTATION](https://zulip.com/help/share-and-upload-files)
*These icons are not this color in the app, but seemed better to be consistent with this formatting in the documentation.*
![Screenshot from 2025-06-11 11-53-56](https://github.com/user-attachments/assets/4c6fa6b2-48f9-43bf-a03b-5467d449b54d)

[CURRENT DOCUMENTATION](https://zulip.com/help/dark-theme)
![Screenshot from 2025-06-11 11-55-09](https://github.com/user-attachments/assets/799e1ff2-1770-4639-9a26-567e6e9b5658)

[CURRENT DOCUMENTATION](https://zulip.com/help/delete-a-message#delete-message-content)
![Screenshot from 2025-06-11 11-56-42](https://github.com/user-attachments/assets/254961c0-4f4a-4f36-b6ad-f5521b09bb4b)

[CURRENT DOCUMENTATION](https://zulip.com/help/starting-a-new-direct-message)
*These instructions are shared on a few help articles.*
![Screenshot from 2025-06-11 11-58-16](https://github.com/user-attachments/assets/70f7f766-36c7-4dbf-b3a4-5bc797c1b2d0)

[CURRENT DOCUMENTATION](https://zulip.com/help/direct-messages#view-your-dms)
![Screenshot from 2025-06-11 12-01-09](https://github.com/user-attachments/assets/ad180138-517b-4091-844d-dee813a6a6bb)

[CURRENT DOCUMENTATION](https://zulip.com/help/edit-a-message)
![Screenshot from 2025-06-11 12-02-43](https://github.com/user-attachments/assets/a546e074-8a0e-4949-b703-5112509d743a)

[CURRENT DOCUMENTATION](https://zulip.com/help/emoji-reactions)
![Screenshot from 2025-06-11 12-03-28](https://github.com/user-attachments/assets/c6a7bb45-bfe0-4e59-bad4-495a4e7f06fa)

[CURRENT DOCUMENTATION](https://zulip.com/help/combined-feed)
![Screenshot from 2025-06-11 12-04-10](https://github.com/user-attachments/assets/88f40ea7-8ab4-48f3-8075-68fbddfcc502)

[CURRENT DOCUMENTATION](https://zulip.com/help/inbox)
![Screenshot from 2025-06-11 12-05-11](https://github.com/user-attachments/assets/1db14c96-9bce-446d-b2e1-290e65673b91)

[CURRENT DOCUMENTATION](https://zulip.com/help/introduction-to-topics#how-to-start-a-new-topic)
*These instructions are shared in a few articles.*
![Screenshot from 2025-06-11 12-06-36](https://github.com/user-attachments/assets/502bbdfd-967a-4195-a30c-43223e0fa287)

[CURRENT DOCUMENTATION](https://zulip.com/help/view-your-mentions)
![Screenshot from 2025-06-11 12-07-29](https://github.com/user-attachments/assets/26fe8e32-cb98-4776-b534-4a366bbce70e)

[CURRENT DOCUMENTATION](https://zulip.com/help/star-a-message)
![Screenshot from 2025-06-11 12-08-20](https://github.com/user-attachments/assets/f616cfe0-1abf-4195-a8eb-b1016611711a)

[CURRENT DOCUMENTATION](https://zulip.com/help/logging-in)
![Screenshot from 2025-06-11 12-09-31](https://github.com/user-attachments/assets/7d57bd16-1179-4e09-8a9b-ac7b561c8bf3)

[CURRENT DOCUMENTATION](https://zulip.com/help/logging-out)
![Screenshot from 2025-06-11 12-10-13](https://github.com/user-attachments/assets/81a7b36f-3bac-4825-9105-9141fa4694e9)

[CURRENT DOCUMENTATION](https://zulip.com/help/marking-messages-as-read#mark-all-messages-as-read)
![Screenshot from 2025-06-11 12-11-21](https://github.com/user-attachments/assets/a3e8884b-e99d-4c40-88df-c9cf9c335cad)

[CURRENT DOCUMENTATION](https://zulip.com/help/mute-a-topic#manage-configured-topics)
![Screenshot from 2025-06-11 12-13-15](https://github.com/user-attachments/assets/6849ad09-c25f-44a8-92b3-2eb856e4a5e7)

[CURRENT DOCUMENTATION](https://zulip.com/help/quote-or-forward-a-message)
*I edited this one a bit more with what I experienced as I used the Flutter app.*
![Screenshot from 2025-06-13 20-31-41](https://github.com/user-attachments/assets/732243df-44d6-4675-ac1e-4de183e7fdfe)

[CURRENT DOCUMENTATION](https://zulip.com/help/resolve-a-topic)
*These changes seem to apply to the React Native app as well...*
![Screenshot from 2025-06-11 12-16-32](https://github.com/user-attachments/assets/0cbaa536-54b2-4fbb-a416-f22528236e24)

[CURRENT DOCUMENTATION](https://zulip.com/help/view-images-and-videos)
*This icon looked a bit different in the mobile app, but seemed fine to use as is. But we could add another mobile specific icon here.*
![Screenshot from 2025-06-11 12-18-04](https://github.com/user-attachments/assets/fa3597fa-ffed-46fc-93ab-79274ce31e49)

[CURRENT DOCUMENTATION](https://zulip.com/help/view-someones-profile)
*I'm pretty sure these are the only things show in the profile view in the Flutter app currently.*
![Screenshot from 2025-06-11 12-19-55](https://github.com/user-attachments/assets/b17a2b28-f74f-4ac5-b951-58ccee4bc75d)

[CURRENT DOCUMENTATION](https://zulip.com/help/view-the-markdown-source-of-a-message)
![Screenshot from 2025-06-11 12-21-04](https://github.com/user-attachments/assets/31fe82d4-44fa-4bc5-b5f3-00529026ea64)

alya commented on help/include/inbox-instructions.md:
```diff
@@ -8,32 +8,22 @@
 !!! tip ""
 
-    You can collapse or expand the list of topics in a channel by tapping the
+    You can collapse or expand the list of topics in a channel by clicking the
```
Hm, why tapping -> clicking here?

laurynmm replied:
Since the icons for collapsing and expanding these are the same in the web/desktop and mobile apps now, I pulled the tip out of the tabbed instructions to be a general tip. But there is that verb difference of tap/click between how we write the documentation for mobile vs web/desktop.

![Screenshot from 2025-06-11 12-05-11](https://github.com/user-attachments/assets/65e78d8f-b49c-425a-876f-1d1f90b8f581)

laurynmm commented:
**Updates**:
- Updated changes to "Quote or forward message" for changes in https://github.com/zulip/zulip-flutter/pull/1575.

alya commented:
While we're here, could you please fix incorrect option naming in the inbox filtering instructions?

![Screenshot 2025-06-13 at 15 13 30](https://github.com/user-attachments/assets/48ee476a-217f-4e01-9bd7-ffc899dd49de)

alya commented on help/quote-or-forward-a-message.md:
```diff
@@ -77,15 +80,20 @@ unnecessarily mentioning someone twice.
 
+1. If you're not in the channel view of the message you want to forward,
+   tap the channel name in the message feed or tap the **Channel feed**
+   <i class="zulip-icon zulip-icon-all-messages mobile-help"></i> icon
+   in the bar at the top of the app.
+
 {!message-long-press-menu.md!}
 
 1. Tap **Quote message**.
 
 1. *(optional)* Delete any parts of the quoted message that you don't want to
    forward.
 
-1. If you are in a channel view, choose a destination topic by tapping the
-   compose box and selecting an existing topic or typing a new topic name.
+1. Choose a destination topic by tapping the compose box and selecting an
+   existing topic or typing a new topic name.
```
I don't feel strongly either way. I think it's reasonable to drop it.

laurynmm commented:
**Updates**:
- Made updates for review feedback above. Specifically these articles:
  - `/dark-theme`: Clarified the text for the default setting in mobile and web/desktop apps
  - `/send-dm`: Changed "Next" to "Compose" and separated out that step in the instructions
  - `/quote-or-forward-a-message`: Deleted the mobile instructions for "Forward message" and added a tip to the "Quote message" mobile instructions for changing the topic.
  - `/view-images-and-videos`: Updated the note about the copy icon actions.
  - `/view-someones-profile`: Added that the mobile app shows the user's role.
  - `/inbox`: Fixed the web app options for filtering by topic status.
- I didn't update `/logging-in` for this comment. The current changes mirror what we have documented for the React Native app. I feel like revising that section further could be a follow-up once we discuss on CZO what we want to do.
- I added a final commit that adds back in the instructions for changing the setting for marking as read on scroll automatically.

**Note that the failing CI is for the blog post link that was added to "View Zulip version" in #34889.**

---

**Updated screenshots**:
*Note the the screenshots for these articles have not been updated in the main PR comment above.*

[CURRENT DOCUMENTATION](https://zulip.com/help/inbox)
![Screenshot from 2025-06-16 15-01-07](https://github.com/user-attachments/assets/410140b2-1e92-4c08-bcdd-0af305107981)

[CURRENT DOCUMENTATION](https://zulip.com/help/view-someones-profile)
![Screenshot from 2025-06-16 14-53-41](https://github.com/user-attachments/assets/4e4c51af-d72a-4284-9c35-41dea0bda8f2)

[CURRENT DOCUMENTATION](https://zulip.com/help/view-images-and-videos)
![Screenshot from 2025-06-16 14-51-32](https://github.com/user-attachments/assets/fbab64d8-5691-482c-9795-c36b8026ec10)

[CURRENT DOCUMENTATION](https://zulip.com/help/quote-or-forward-a-message)
![Screenshot from 2025-06-16 14-00-31](https://github.com/user-attachments/assets/460593d1-1d18-43ff-8e19-3fb9b4b70b7c)

[CURRENT DOCUMENTATION](https://zulip.com/help/starting-a-new-direct-message)
![Screenshot from 2025-06-16 13-41-57](https://github.com/user-attachments/assets/98bf2a0e-e687-47db-ad95-67a6d0cd9be5)

[CURRENT DOCUMENTATION](https://zulip.com/help/dark-theme)
![Screenshot from 2025-06-16 13-35-06](https://github.com/user-attachments/assets/0bf83206-6ffa-429e-b971-d5fe80ec3e00)

[CURRENT DOCUMENTATION](https://zulip.com/help/marking-messages-as-read)
![Screenshot from 2025-06-16 12-51-46](https://github.com/user-attachments/assets/5a451798-4b00-43b0-896b-0ade331c611e)
